prometheus/manifests/prometheus/010-prometheus-cr.yaml

```diff
@@ -9,16 +9,22 @@ spec:
     requests:
       memory: 400Mi
   serviceAccountName: prometheus
-  serviceMonitorNamespaceSelector: {}
-  serviceMonitorSelector: {}
-#  securityContext:
-#    fsGroup: 2000
-#    runAsNonRoot: true
-#    runAsUser: 1000
-#  storage:
-#    volumeClaimTemplate:
-#      spec:
-#        storageClassName: fast
-#        resources:
-#          requests:
-#            storage: 10Gi
+  serviceMonitorSelector:
+    matchLabels:
+      app: storageos
+  securityContext:
+    fsGroup: 2000
+    runAsNonRoot: true
+    runAsUser: 1000
+  storage:
+    volumeClaimTemplate:
+      metadata:
+        name: data
+        labels:
+          env: prod
+      spec:
+        accessModes: ["ReadWriteOnce"]
+        storageClassName: fast
```

Comment:
I'm not sure, but do we want to tell people to use a StorageClass with replication enabled, rather than fast?
@avestuk

Reply:
Running with a replica enabled is safer so I think it's not a bad idea.